lib/krane/render_task.rb

```diff
@@ -12,13 +12,13 @@ class RenderTask
     #
     # @param logger [Object] Logger object (defaults to an instance of Krane::FormattedLogger)
     # @param current_sha [String] The SHA of the commit
-    # @param template_dir [String] Path to a directory with templates to render (deprecated)
-    # @param template_paths [Array<String>] An array of template paths to render
+    # @param template_dir [String] Path to a directory with templates to render (*deprecated*)
```

Comment:
Are you 🔥 this in a different PR?

Reply:
Yes, on https://github.com/Shopify/kubernetes-deploy/pull/622